docs/ske/10-integrations/10-backstage/03-generator.mdx

```diff
@@ -270,3 +285,151 @@ Resource object itself:
+
+### Generate a single Backstage catalog
+
+Using the SKE Backstage Component Promise, you can optionally choose to write a single
+`catalog-info.yaml` file for Backstage to read from, instead of the default behaviour of
+one catalog file per Promise or resource request.
+
+To do this, there are two steps required:
+- Configuring the SKE Backstage Generator to write all of the Backstage manifests in a
+single `catalog-info.yaml`.
+- Configuring your Promise to use the SKE Backstage Component Promise.
+
+First, apply the following config map to your Platform cluster:
+
+```yaml
+apiVersion: v1
+data:
+  allInOne: "true"
+kind: ConfigMap
+metadata:
+  name: ske-backstage-component-promise-config
+  namespace: kratix-platform-system
+```
+
+The `allInOne` field will be read by the SKE Backstage Component Promise, which will then
+aggregate the Backstage Component and Template files for all Promises and Resources into
+a single `catalog-info.yaml` file in the Backstage Destination.
+
+Next, configure the SKE Backstage Generator to send a request to the SKE Backstage
+Component Promise in the Resource and Promise workflows:
+
+```yaml
+  # ...
+  workflows:
+    resource:
+      configure:
+      - apiVersion: platform.kratix.io/v1alpha1
+        kind: Pipeline
+        metadata:
+          name: resource-configure
+          namespace: default
+        spec:
+          #highlight-start
+          rbac:
+            permissions:
+              - apiGroups: ["syntasso.io"]
+                verbs: ["*"]
+                resources: ["backstagecomponents"]
+          #highlight-end
+          containers:
+          - image: ghcr.io/syntasso/kratix-marketplace/jenkins-configure-pipeline:v0.1.0
+            name: jenkins-resource-pipeline
+          - image: ghcr.io/syntasso/ske-backstage-generator:v0.3.0
+            name: ske-backstage-generator
+          #highlight-start
+            args:
+            - --make-request
+          #highlight-end
+    promise:
+      configure:
+      - apiVersion: platform.kratix.io/v1alpha1
+        kind: Pipeline
+        metadata:
+          name: promise-configure
+          namespace: default
+        spec:
+          #highlight-start
+          rbac:
+            permissions:
+              - apiGroups: ["syntasso.io"]
+                verbs: ["*"]
+                resources: ["backstagecomponents"]
+          #highlight-end
+          containers:
+          - image: ghcr.io/syntasso/kratix-marketplace/jenkins-configure-pipeline:v0.1.0
+            name: jenkins-promise-pipeline
+          - image: ghcr.io/syntasso/ske-backstage-generator:v0.3.0
+            name: ske-backstage-generator
+          #highlight-start
+            args:
+            - --make-request
+          #highlight-end
+```
+
+With these changes, your SKE Backstage integration will result in a single
+`catalog-info.yaml` file being written to your Backstage Destination containing all of the
+generated Component and Template manifests.
```

Comment:
```suggestion
generated Component and Template manifests. You must ensure this change is mirrored across all Promises using the ske-backsage-generator, otherwise multiple `catalog-info.yaml` files will exist
```

Reply:
![image](https://github.com/user-attachments/assets/2aa89682-2fb6-481c-813f-17df8bbb8171)

Reply:
![image](https://github.com/user-attachments/assets/d68e91ac-84e7-4607-b8ec-2eae3b74163a)